src/test/java/org/kohsuke/github/LifecycleTest.java

```diff
@@ -45,18 +45,20 @@ public void testCreateRepository() throws IOException {
 
     private void updateAsset(GHRelease release, GHAsset asset) throws IOException {
         asset.setLabel("test label");
-        assertEquals("test label", release.getAssets().get(0).getLabel());
+        assertEquals("test label", release.fetchAssets().get(0).getLabel());
```

Comment:
Keep the test pointing to the deprecated method since it wraps the new method and we want to show the behavior hasn't changed. 

```suggestion
        assertEquals("test label", release.getAssets().get(0).getLabel());
```